### ISSUE TITLE
Please put license in the source code

### ISSUE THREAD

**Issue by ian-kelling** [lazierthanthou/sqlite-manager#61]
I'd like to add this project to the free software directory, https://directory.fsf.org/wiki/Main_Page, but the only reference to the license in the source code is in some metadata. Please add the mpl to a file like COPYING, and put license headers in the source code.

From https://www.mozilla.org/en-US/MPL/1.1/:

"3.5. Required Notices.

You must duplicate the notice in Exhibit A in each file of the Source Code"

https://www.mozilla.org/en-US/MPL/1.1/#exhibit-a

**Comment by victorwestmann** [comment]
+1 for this.

**Comment by Croydon** [comment]
@lazierthanthou Please do this

**Comment by justinclift** [comment]
Well, this project seems to be effectively dead, so it's not super likely it'll receive further development attention.

**Comment by Croydon** [comment]
Yes, but adding a license is really no big afford, while at the same time providing much more clearness.

**Comment by justinclift** [comment]
Maybe make a PR with the license, so it's as minimal developer effort as possible needed? :smile:
